server/controller/src/main/java/ai/starwhale/mlops/api/DatasetApi.java

```diff
@@ -235,8 +232,8 @@ void pullDs(
             @PathVariable(name = "projectUrl") String projectUrl,
             @PathVariable(name = "datasetUrl") String datasetUrl,
             @PathVariable(name = "versionUrl") String versionUrl,
-            @Parameter(name = "part_name", description = "optional, _manifest.yaml is used if not specified")
-            @RequestParam(name = "part_name", required = false) String partName,
+            @Parameter(name = "signature", description = "optional, _manifest.yaml is used if not specified")
```

Comment:
Not always signature is passed such as _manifest.yaml. Indeed , when it appears like a signature, it is a file who's name is the signature

Reply:
fixed